app/i18n/it/index.php

```diff
@@ -14,6 +14,16 @@
 	'about' => array(
 		'_' => 'Informazioni',
 		'agpl3' => '<a href="https://www.gnu.org/licenses/agpl-3.0.html">AGPL 3</a>',	// IGNORE
+		'bug_reports' => array(
+			'environment_information' => array(
+				'_' => 'Environment information',	// TODO
```

Comment:
```suggestion
				'_' => 'Informazioni sul sistema',
```
@math-GH I had to change the sentence a little bit to keep the sense and translated it this way `System information`

Reply:
@Frenzie : What do you think? What is a better choice for the English term? `Environment information` or `System information`?

Reply:
I'd go with system information since it's widely used for the purpose. 

Incidentally, I'd add `php_uname()`, probably limited to s, r, v, and m (because hostname is a bit sensitive). There've been too many arm-exclusive issues not to.

```php
php_uname('s') . ' ' . php_uname('r') . ' ' . php_uname('v') . ' ' . php_uname('m')
```

https://www.php.net/manual/en/function.php-uname.php

Reply:
> Incidentally, I'd add `php_uname()`, probably limited to s, r, v, and m (because hostname is a bit sensitive). There've been too many arm-exclusive issues not to.
> 
> ```
> php_uname('s') . ' ' . php_uname('r') . ' ' . php_uname('v') . ' ' . php_uname('m')
> ```
> 
> https://www.php.net/manual/en/function.php-uname.php

Do you mean it is better than `$_SERVER['SERVER_SOFTWARE']`?